geom.js

```diff
@@ -2,31 +2,94 @@ class Rectangle {
   constructor(length, width) {
     this.length = length;
     this.width = width;
+    console.log(this.length + " and " +this.width);
+  }
+  isSquare(){
+    if (this.length==this.width)
+    {
+      console.log("Yes it is square");
+      return true;
+    }
+    else{
+      console.log("It is not a square?");
+    }
+  }
+  area(){
+    let area1= this.length * this.width;
```

Comment:
You should return this value so that it can be used whenever somebody invokes this function!